ansible/roles/common/tasks/main.yml

```diff
@@ -66,8 +66,17 @@
 - name: Install common packages for RHEL 9
   when:
   - ansible_distribution_major_version == "9"
```

Comment:
Should we add a condition for RHEL?
Like "debian 9" would trigger the execution of those tasks too?